examples/00-EDB/06_Advanced_EDB.py

```diff
@@ -1,49 +1,44 @@
 """
-EDB: parametric via creation
-----------------------------
+# EDB: parametric via creation
+
 This example shows how you can use EDB to create a layout.
```

Comment:
All libraries use "shows how to use" rather than "shows how you can use". I recommend that you do a search and replace to use the more standard terminology.